bin/build

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 
+# link references
+CPAC_DIRECTORY=$(python -c "import os, CPAC; print(os.path.abspath(os.path.dirname(CPAC.__file__)))")
+ln ${CPAC_DIRECTORY}/resources/configs/1.7-1.8-nesting-mappings.yml docs/_sources/references/1.7-1.8-nesting-mappings.yml || true
+ln ${CPAC_DIRECTORY}/resources/configs/1.7-1.8-deprecations.yml docs/_sources/references/1.7-1.8-deprecations.yml || true
+ln ${CPAC_DIRECTORY}/resources/configs/default_pipeline.yml docs/_sources/references/default_pipeline.yml || true
```

Comment:
default_pipeline.yml will be in `${CPAC_DIRECTORY}/../dev/docker_data/default_pipeline.yml`

Reply:
I think `${CPAC_DIRECTORY}/..` will only work if CPAC is installed with `-e, --editable`, otherwise the source code isn't included in the installation.

In 1.8, the installation process for CPAC copies that file from `dev/docker/data` into `resources/configs`:
https://github.com/FCP-INDI/C-PAC/blob/0926b451dd8622b25eb68c7bcc770f0156238b23/setup.py#L48-L50

You're right that https://github.com/FCP-INDI/fcp-indi.github.com/blob/42712c93affe60bff76570e0bdabf0d39622c779/bin/build#L7 wouldn't work for C-PAC < v1.8.0. I [added "(C-PAC ≥ v1.8.0)" to the "Build locally" header](https://github.com/FCP-INDI/fcp-indi.github.com/blob/d5a668eae9bacb771812fb0059696de93f5eb0a9/CONTRIBUTING.md#build-locally-c-pac-%E2%89%A5-v180).